v2/elasticsearch-common/src/main/java/com/google/cloud/teleport/v2/elasticsearch/options/ElasticsearchWriteOptions.java

```diff
@@ -211,7 +211,7 @@ public interface ElasticsearchWriteOptions extends PipelineOptions {
       optional = true,
       description = "UDF JavaScript Function Name for isDelete",
       helpText =
-          "UDF JavaScript function Name for function that will determine if document should be deleted rather than inserted or updated. The function should return string value \"true\" or \"false\". Default: none")
+          "UDF JavaScript function name for function that will determine if document should be deleted rather than inserted or updated. The function should return string value \`true\` or \`false\`. Default: none.")
```

Comment:
```suggestion
          "The name of the UDF JavaScript function that determines whether to delete the document instead of inserting or updating it. The function returns a string value of \`true\` or \`false\`. Defaults to: none.")
```